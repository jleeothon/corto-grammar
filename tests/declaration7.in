list{int{WIDTH_8}} a
