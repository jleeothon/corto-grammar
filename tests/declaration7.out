(program (statement (declaration (typeLabel (anonymousType (identifier list) (initializerBraces { (initializerList (initializerElem (initializerValue (anonymousType (identifier int) (initializerBraces { (initializerList (initializerElem (initializerValue (identifier WIDTH_8)))) }))))) }))) a) \n))
